src/backend/distributed/commands/multi_copy.c

```diff
@@ -852,6 +864,34 @@ CanUseBinaryCopyFormat(TupleDesc tupleDescription, CopyOutState rowOutputState)
 
 
 /*
+ * BinaryOutputFunctionIsDefined checks whether binary output function is defined
+ * for the given type.
+ */
+static bool
+BinaryOutputFunctionIsDefined(Oid typeId)
+{
+	HeapTuple typeTuple;
+	Form_pg_type pt;
```

Comment:
We should initialize this and the variable above. Also we need more descriptive name than pt.